cypress/e2e/user-flows/scenario_008_edit_mock_from_simulator.cy.js

```diff
@@ -58,7 +58,7 @@ describe("Scenario 008", () => {
     slice.openSimulator();
 
     // Wait for the editor to be fully loaded
-    cy.contains("label > span", "Title").should("be.visible");
+    editor.contains("Title").should("be.visible");
```

Comment:
editor.root.contains would have worked also, to avoid creating contains for each page objects